gravitee-apim-console-webui/src/management/settings/groups/group/add-members-dialog/add-members-dialog.component.ts

```diff
@@ -0,0 +1,228 @@
+ */
+
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogModule, MatDialogRef } from '@angular/material/dialog';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { MatIconModule } from '@angular/material/icon';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatSelectModule } from '@angular/material/select';
+import { MatInputModule } from '@angular/material/input';
+import { MatButtonModule } from '@angular/material/button';
+import { CommonModule } from '@angular/common';
+import { debounceTime, distinctUntilChanged, map, startWith } from 'rxjs/operators';
+import { MatAutocompleteModule, MatAutocompleteSelectedEvent } from '@angular/material/autocomplete';
+import { Observable, of, switchMap } from 'rxjs';
+import { MatListModule } from '@angular/material/list';
+import { MatChipsModule } from '@angular/material/chips';
+
+import { GioPermissionService } from '../../../../../shared/components/gio-permission/gio-permission.service';
+import { GroupService } from '../../../../../services-ngx/group.service';
+import { SnackBarService } from '../../../../../services-ngx/snack-bar.service';
+import { Group } from '../../../../../entities/group/group';
+import { Role } from '../../../../../entities/role/role';
+import { ApiPrimaryOwnerMode } from '../../../../../services/apiPrimaryOwnerMode.service';
+import { EnvironmentSettingsService } from '../../../../../services-ngx/environment-settings.service';
+import { SearchableUser } from '../../../../../entities/user/searchableUser';
+import { Member } from '../../../../../entities/management-api-v2';
+import { GroupMembership } from '../../../../../entities/group/groupMember';
+import { RoleName } from '../membershipState';
+import { UsersService } from '../../../../../services-ngx/users.service';
+
+@Component({
+  selector: 'add-member-dialog',
+  standalone: true,
+  imports: [
+    CommonModule,
+    MatDialogModule,
+    ReactiveFormsModule,
+    MatIconModule,
+    MatInputModule,
+    MatButtonModule,
+    MatFormFieldModule,
+    MatSelectModule,
+    MatAutocompleteModule,
+    MatListModule,
+    MatChipsModule,
+  ],
+  templateUrl: './add-members-dialog.component.html',
+  styleUrl: './add-members-dialog.component.scss',
+})
+export class AddMembersDialogComponent implements OnInit {
+  defaultAPIRoles: Role[];
+  defaultApplicationRoles: Role[];
+  defaultIntegrationRoles: Role[];
+  addMemberForm: FormGroup<{
+    defaultAPIRole: FormControl<string>;
+    defaultApplicationRole: FormControl<string>;
+    defaultIntegrationRole: FormControl<string>;
+    searchTerm: FormControl<string>;
+  }>;
+  filteredUsers$: Observable<SearchableUser[]>;
+  selectedUsers: SearchableUser[] = [];
+
+  private group: Group;
+  private memberships: GroupMembership[] = [];
+  private members: Member[] = [];
+
+  constructor(
+    @Inject(MAT_DIALOG_DATA) public data: any,
```

Comment:
What about type all dialog with like this : https://github.com/gravitee-io/gravitee-api-management/blob/d2256d6b62315eb3a0189fbcaf973d52d68b6ff5/gravitee-apim-console-webui/src/management/api/api-navigation/api-confirm-deployment-dialog/api-confirm-deployment-dialog.component.ts#L29

event if the result it's a void. 
And also always specify type with open method like 
```
this.matDialog
                      .open<ApiConfirmDeploymentDialogComponent, ApiConfirmDeploymentDialogData, ApiConfirmDeploymentDialogResult>(
                        ApiConfirmDeploymentDialogComponent,
                        ...
```